packages/components/src/Layout/OneColumn/OneColumn.component.js

```diff
@@ -11,7 +11,7 @@ import WithDrawer from '../../WithDrawer';
  * @example
  <OneColumn name="Hello world"></OneColumn>
  */
-function OneColumn({ drawers, children, tabs, ...props }) {
+function OneColumn({ drawers, children, tabs, injected, ...props }) {
```

Comment:
Don't you need to rename `injected` to `inject` since you changed it on `Layout`?